src/sugar3/power.py

```diff
@@ -47,9 +91,18 @@ def __del__(self):
         self._remove_flag_file()
 
     def suspend_breaks_collaboration(self):
+        """
+        Does automatic idle suspend break collaboration with this
+        toolkit?  Yes.  For future use by a toolkit with more
+        resilient collaboration.
+
+        Returns:
+            True
+        """
         return True
 
     def inhibit_suspend(self):
+        """Inhibit automatic idle suspend until restored."""
```

Comment:
We also leave a newline here; as seen in
https://github.com/sugarlabs/sugar-toolkit-gtk3/blob/master/src/sugar3/graphics/xocolor.py#L279
https://github.com/sugarlabs/sugar-toolkit-gtk3/blob/master/src/sugar3/graphics/toolbutton.py#L206
etc